jlexer/lexer.go

```diff
@@ -667,6 +667,8 @@ func (r *Lexer) String() string {
 	}
 	var ret string
 	if r.token.byteValueCloned {
+		r.token.byteValueCloned = false
```

Comment:
Any details on why it fixes the issue?

Actually better to provide details why the issue occurs at all... byteValueCloned should be TRUE only iff the value was allocated and no longer refers to original buffer. So this TRUE path should be safe. Why it turns out it is not?